.buildkite/init-settlements.yml

```diff
@@ -2,49 +2,47 @@ agents:
   queue: "snapshots"
 
 steps:
-  # epoch number provided in env or prompted
+  # epoch number provided in ENV and then waiting for confirmation to proceed
+  # or waiting to be prompted by the user after notification is sent
   - input: "What is the epoch to init settlements for?"
     fields:
       - text: "Epoch"
         key: "epoch"
         format: "[0-9]+"
     if: "build.env('EPOCH') == null"
-
-  - command: "echo 'Epoch is: $$EPOCH'"
-    if: "build.env('EPOCH') != null"
-  - block: "Confirm to get processed init settlements"
-    prompt: "Please review the 'Unblock' if ready to proceed."
+  - command: "echo 'Epoch is: $EPOCH'"
     if: "build.env('EPOCH') != null"
 
-  - label: ":hammer_and_wrench: :rust: Build"
-    commands:
-    - '. "$HOME/.cargo/env"'
-    - 'cargo build --release --bin init-settlement'
-    artifact_paths:
-      - target/release/init-settlement
-
   - wait: ~
 
   - label: ":mega: Notification initializing settlements"
     commands:
+    - 'epoch=${EPOCH:-$(buildkite-agent meta-data get epoch)}'
+    - 'buildkite-agent meta-data set epoch "$$epoch"'
+    - 'echo "Epoch: $$epoch"'
     - |
-      epoch=${EPOCH:-$(buildkite-agent meta-data get epoch)}
-      curl "$$DISCORD_WEBHOOK_VALIDATOR_BONDS" -H "Content-Type: application/json" -d '{
+      echo "$$DISCORD_WEBHOOK_VALIDATOR_BONDS" -H "Content-Type: application/json" -d '{
         "embeds": [
           {
-            "title": "Initializing Settlements for Validator Bonds ('"$$epoch"').",
+            "title": "Confirm Settlements Initialization for Validator Bonds ('"$$epoch"').",
             "url": "'"$$BUILDKITE_BUILD_URL"'",
             "color": "8388863"
           }
         ]
       }'
 
-  - label: ":gear: Setup"
+  - block: "Confirm to get processed init settlements"
+    prompt: "Please review the 'Unblock' if ready to proceed."
+    if: "build.env('EPOCH') != null"
```

Comment:
```suggestion
```
I think this is just a copy-paste error, right?

Reply:
No, it was meant to get blocked processing when init settlements `EPOCH` env variable is provided, i.e., in case of the other pipeline triggers this one.

When the `EPOCH` env is not provided, i.e., this is executed by hand from buildkite, then there is prompt instead of that where the user has to provide the epoch number.

Reply:
ah - otherwise it would be obtained from `buildkite-agent meta-data get epoch`, i see